tsconfig.json

```diff
@@ -29,16 +29,21 @@
     // "reactNamespace": "",                             /* Specify the object invoked for 'createElement'. This only applies when targeting 'react' JSX emit. */
     // "noLib": true,                                    /* Disable including any library files, including the default lib.d.ts. */
     // "useDefineForClassFields": true,                  /* Emit ECMAScript-standard-compliant class fields. */
-    "moduleDetection": "force" /* Control what method is used to detect module-format JS files. */ /* Modules */,
-    "module": "esnext" /* Specify what module code is generated. */,
-    // "rootDir": "./src/" /* Specify the root folder within your source files. */,
-    "moduleResolution": "node" /* Specify how TypeScript looks up a file from a given module specifier. */, // "baseUrl": "./",                                  /* Specify the base directory to resolve non-relative module names. */
+    "moduleDetection": "force"                           /* Control what method is used to detect module-format JS files. */ /* Modules */,
+    "module": "ESNext"                                   /* Specify what module code is generated. */,
+    "moduleResolution": "node"                           /* Specify how TypeScript looks up a file from a given module specifier. */,
+    // "rootDir": "./src/"                               /* Specify the root folder within your source files. */,
+    "baseUrl": "./",                                     /* Specify the base directory to resolve non-relative module names. */
```

Comment:
Also I think part of the other PR? But also a good thing to test on whether/how it affects our cronjob scripts.

Reply:
These changes were reverted.